script/Arcueid/Day 7/07_00_ARC07_3D.txt

```diff
@@ -77,7 +78,7 @@ It was the sort of stare that made me feel as though she'd swoop down on me the
 -- Page 233, Offset 17138.
 -- ヴローヴは死んだ。なのに、どうしてまだ吸血鬼騒ぎが起こる？　まさかとは思うけど、あれは、おまえが―――」
 // Lots of strong pauses in VA, here ...
-Vlov is dead. So then, why are these vampire incidents still happening?
+Vlov is dead. So then, why are these vampiric incidents still happening?
```

Comment:
We used "killings" rather than "incidents" in 7_1D but it was for 事件, not 騒ぎ which is more like "vampiric disturbances" like they're going around and causing a big racket. I think it's a little too close to him saying "vampiric nonsense" to promote this to "killings"